<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinAdminShowUser</class>
 <widget class="QMainWindow" name="WinAdminShowUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>691</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Information du compte</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>621</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditNom">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Prenom: </string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPrenom">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>401</width>
       <height>37</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonAfficher">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Afficher</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>691</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>829</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
